module/authentication/src/Application/Security/User/User.php

```diff
@@ -9,10 +9,13 @@
 
 namespace Ergonode\Authentication\Application\Security\User;
 
-use Ergonode\Account\Domain\Entity\UserInterface;
+use Ergonode\Core\Domain\User\UserInterface;
 
 class User implements UserInterface
 {
+    /**
+     * Allows easy serialization into token.
+     */
```

Comment:
:1st_place_medal:

Reply:
Yup... started to wonder why I added this... so not very obvious 😂